src/main/java/com/skypro/employee/DepartamentController.java

```diff
@@ -0,0 +1,39 @@
+package com.skypro.employee;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Map;
+
+@RequestMapping("/department")
+@RestController
+public class DepartamentController {
+    private final DepartmentService serviceDep;
```

Comment:
```suggestion
    private final DepartmentServiceInterface serviceDep;
```
инъекцию надо делать по интерфейсу